src/librustc_hir/hir.rs

```diff
@@ -2666,7 +2666,7 @@ pub type TraitMap<ID = HirId> = NodeMap<Vec<TraitCandidate<ID>>>;
 
 // Map from the NodeId of a glob import to a list of items which are actually
 // imported.
-pub type GlobMap = NodeMap<FxHashSet<Symbol>>;
+pub type GlobMap = FxHashMap<LocalDefId, FxHashSet<Symbol>>;
```

Comment:
Could you remove this?
This alias is used in a single place and was mostly obscuring the logic rather than helping.

Reply:
Done